config/resources/domain.json

```diff
@@ -265,58 +328,99 @@
         "temporal": {
           "type": "boolean",
           "predicate": "ext:temporal"
```

Comment:
**Suggestion:** Remove the `temporal` definition (migrate in `variable-signage`).